python/mxnet/image.py

```diff
@@ -101,7 +101,57 @@ def random_crop(src, size, interp=2):
 
 
 def center_crop(src, size, interp=2):
-    """Centrally crop src with size. Upsample result if src is smaller than size."""
+    """Crops the image `src` to the given `size` by trimming all the four sides
+    while retaining the center of the image. Upsamples if `src` is smaller than
+    `size`.
+
+    Note: This requires OpenCV to be installed.
```

Comment:
requires mxnet to be compiled with USE_OPENCV.
also this not the right format for Note